itou/templates/static/legal/privacy.html

```diff
@@ -131,26 +147,26 @@ <h2 class="mt-3 mt-lg-5" id="quels-droits-avez-vous-">Quels droits avez-vous ?</
 
                     <h2 class="mt-3 mt-lg-5" id="qui-va-avoir-accès-à-ces-données-">Qui va avoir accès à ces données ?</h2>
                     <p>
-                        Les accès aux données sont strictement encadrés et juridiquement justifiés. Les personnes suivantes vont avoir accès aux données : (liste à modifier selon les personnes ayant accès aux données)
+                        Les accès aux données sont strictement encadrés et juridiquement justifiés. Les personnes suivantes vont avoir accès aux données :
                     </p>
                     <ul>
                         <li>
-                            Les membres des emplois de l’inclusion qui ont besoin des données dans leurs missions ou qui y ont accès de fait (développeur, etc.) ;
+                            Les membres des Emplois de l’inclusion qui ont besoin des données dans leurs missions ou qui y ont accès de fait (développeur, etc.) ;
                         </li>
-                        <li>Pôle emploi ;</li>
+                        <li>Pôle Emploi ;</li>
```

Comment:
Pas de majuscule à "emploi"
```suggestion
                        <li>Pôle emploi ;</li>
```

Reply:
Moi j'ai suivi le document fourni :man_shrugging: 
Je corrige!